drizzlepac/devutils/search_skyframes.py

```diff
@@ -131,11 +131,11 @@ def make_search_string(arg_dict):
     """
     search_string = ""
     for item in arg_dict:
-        if not arg_dict[item] or item is "date_range":
+        if not arg_dict[item] or item == "date_range":
```

Comment:
Good catch, these incorrect "is". Makes me nervous that there might be more, so I grepped through the code base and identified some additional possible cases:
photeq.py:                if whdu is ref_hdu:
photeq.py:            if whdu is ref_hdu:
photeq.py:        if h is not _sh_log and isinstance(h, logging.FileHandler):
haputils/pipeline_graphics.py:            if self.Umask is not ma.nomask:

Reply:
those in `photeq` are correct.